credsweeper/__main__.py

```diff
@@ -97,9 +97,15 @@ def get_arguments() -> Namespace:
                         dest="log",
                         metavar="LOG_LEVEL",
                         choices=list(Logger.LEVELS))
+    parser.add_argument("--size_limit",
+                        help="set size limit of files that for scanning (unit: MB)",
```

Comment:
Small notation issue, but we are probably talking about `MiB`, not `MB`

Apparently there is a difference between Megabyte (MB, 1 000 000 bytes) and [Mebibyte](https://simple.wikipedia.org/wiki/Mebibyte) (MiB, 1 048 576 bytes)
Some systems got it confuse, so that's why you may have noticed that file size may change if you upload a file from Windows to Linux or vice versa

This isn't really a big deal, as almost all the software has confused it anyway. I just want to point this out.

Reply:
Fixed with [62c3f76](https://github.com/Samsung/CredSweeper/pull/125/commits/62c3f76463fdd742b79829cd8001e1d232c9021e).

Thank you for review.